contracts/Removal.sol

```diff
@@ -80,57 +110,138 @@ contract Removal is
     return removalId.unpackRemovalIdV0();
   }
 
+  // TODO do we want a batch version of this?
+  /**
+   * @notice Get the restriction schedule id (which is the removal's project id) for a given removal id.
+   */
+  function getProjectIdForRemoval(uint256 removalId)
+    public
+    view
+    returns (uint256)
+  {
+    return _removalIdToProjectId[removalId];
+  }
+
+  /**
+   * @notice Get the restriction schedule data for a given removal id.
+   */
+  function getScheduleDataForRemovalId(uint256 removalId)
```

Comment:
Is this used anymore?

Reply:
I feel like it doesn't hurt to leave this here

Reply:
more code to maintain :shrug: -- ok leaving it for now until I get a better idea as to why both would be needed